fxsql/module.go

```diff
@@ -171,3 +171,16 @@ func RunFxSQLSeeds(names ...string) fx.Option {
 		},
 	)
 }
+
+
+// RunFxSQLSeeds runs database seeds with a context and shutodwn.
+func RunFxSQLSeedsAndShutdown(names ...string) fx.Option {
```

Comment:
we need a test in module_test for this

you can take inspiration [from this one](https://github.com/ankorstore/yokai/blob/main/fxsql/module_test.go#L173)

Reply:
also the doc of this method must match the func name

Reply:
damn, nice catch, sec